clayui.com/plugins/gatsby-plugin-clay-css-tasks/gatsby-node.js

```diff
@@ -105,33 +117,53 @@ const generateFiles = (pluginOptions) => {
 	);
 };
 
+// Compiles and writes `atlas.css` and `base.css` files to `/static/css/`
+
+const generateCSSFiles = (pluginOptions) => {
```

Comment:
@pat270 to have no problem in the production build, we need to make sure to create the file `/static/css/`, because we are considering that it exists below. Instead of doing some node checking, we can create the css folder and add an empty file called `.keep`, which is a common pattern in repositories. For this to work we would have to improve the ignore of `static/css`.

Reply:
I just pushed a change doing some node checking. I'll change that to this.

Reply:
@pat270 it's okay to check this with the node as well.